extensions/commands/src/org/exist/commands/info/Lock.java

```diff
@@ -53,7 +53,8 @@ public void process(XmldbURI collectionURI, String[] commandData) throws Command
 				Collection collection = broker.getCollection(collectionURI);
 
 				out().println("Collection lock:");
-				collection.getLock().debug(out());
+				//TODO:check where that method is
+				//collection.getLock().debug(out());
```

Comment:
What is this change for? Is this still outstanding?

Reply:
The complete class does not compile due no such method exist on the `org.exist.collections.Collection`

Reply:
I see. Thanks